Diana_Mokhnitskaya/HW7/task_1.py

```diff
@@ -0,0 +1,32 @@
+def arithmetic(): ...
```

Comment:
implementation should be here

Reply:
Done 479e502f337dab6375e662abd4d2ecf11048fa6c

Reply:
Where is the fix?